app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,8 +26,20 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left text-gray-400">Title</th>
+            <th className="text-left text-gray-400">Description</th>
+            <th className="text-left text-gray-400">
+              <label>Tags</label>
+            </th>
+            <th className="text-left text-gray-400">
+              <label>Created Date</label>
+            </th>
+            <th className="text-left text-gray-400">
+              <label>Due Date</label>
+            </th>
+            <th className="text-left text-gray-400">
+              <label>Contact</label>
+            </th>
```

Comment:
Why do we need a `label` tag here?

Reply:
Used label tag because , I saw those column as centred in the Figma design. While referring the lib/styles/components/table.scss, found the styling given for label centres it correctly.